.github/workflows/go.yml

```diff
@@ -0,0 +1,130 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+      - release/*
+      # skip non-trunk PMM-XXXX/SAAS-XXXX branch builds, but still build pull requests
+      - '!PMM-[0-9]+'
+      - '!SAAS-[0-9]+'
```

Comment:
But why do we need negative conditions here? If we will drop these lines only `main` and `release` will trigger build, no?

Reply:
it was migrated from travis.yml 
it ignores (probably) only branches with no description in the name. e.g. the current branch has the description.
Do you think this condition is outdated?

Reply:
I think yes, we are using something like this in all other repos: https://github.com/percona/pmm-managed/blob/58e8a4f6b8f5a1a3b5ba4c1817e757eb388c20bc/.github/workflows/ci.yml#L3-L15

Reply:
Ok, I fixed this.